api/src/neo4j/queries/externalDb.js

```diff
@@ -21,6 +33,12 @@ RETURN { externalDb: properties(db), components: COLLECT({ component: c, version
     };
   });
 
+  if (dbName === 'MetabolicAtlas') {
+    externalDb.dbName = 'MetabolicAtlas';
```

Comment:
To reduce the amount of hard-coded strings, I would suggest:
```suggestion
    externalDb.dbName = dbName;
```

Reply:
Addressed by 4c33332d